llvm/docs/CommandGuide/llvm-mustachespec.rst

```diff
@@ -0,0 +1,13 @@
+llvm-mustachespec - LLVM tool to test Mustache Compliance Library
```

Comment:
I'd consider llvm-check-mustachespec or llvm-test-mustachespec to be more explicit.